flags/mod.ts

```diff
@@ -199,12 +237,26 @@ export function parse(
     });
 
     const key = keys[keys.length - 1];
+    const collectable = collect &&
+      (flags.collectAll || !!flags.collect[name]);
+
     if (
-      get(o, key) === undefined ||
-      get(flags.bools, key) ||
-      typeof get(o, key) === "boolean"
+      hasCollect || hasTypes ? !collectable : (
+        get(o, key) === undefined ||
+        get(flags.bools, key) ||
+        typeof get(o, key) === "boolean"
+      )
     ) {
+      // if (hasTypes && get(o, key) !== undefined) {
+      //   throw new Error(
+      //     `Option "${
+      //       name.length > 1 ? "--" : "-"
+      //     }${name}" can be used only once.`,
+      //   );
+      // }
```

Comment:
please remove these lines

Reply:
Or should this be uncommented in the future?

Reply:
It depends on how we want to handle duplicate options. You said you would prefer to throw an error for duplicate options. But currently we have tests to make sure that options can be overwritten.
If we want to keep the current behavior, we can remove those lines.
If we want to throw an error message for duplicate options, we can uncomment them.
If we want to make it configurable, I can add a todo to do that in a separate PR.

Reply:
Current behavior (overwriting) makes enough sense to me. Let's try this as first pass.